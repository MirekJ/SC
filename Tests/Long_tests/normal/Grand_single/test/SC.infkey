6bcc6311-1bd2-0a99-5cc0-2b891f9ae91a
