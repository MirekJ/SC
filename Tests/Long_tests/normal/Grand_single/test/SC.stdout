Using RAN2 random generator

Patchy Spherocylinders version 3.6
-------------------------------------
Reading options...
 Pressure coupling type:                             1
 Pressure:                                           0.00000000
 Replica exchange pressure:                          0.00000000
 Average volume change attempts per sweep:           0.00000000
 Equilibration sweeps:                               0
 Sweeps between step size adjustments:               0
 Production sweeps:                                  30000000
 Sweeps between statistics samples:                  50000
 Sweeps between statistics reports:                  50000
 Average chain move attempts per sweep:              0.00000000
 Initial maximum displacement:                       0.21200000
 Inititial maximum angular change (degrees):         7.50000000
 Inititial maximum angular cone angle (degrees):     0.00000000
 Inititial maximum geo.box edge change:              0.00000000
 Initial maximum chain displacement:                 0.00000000
 Inititial maximum chain angular change (degrees):   0.00000000
 Temperature in kT/e:                                0.24426000
 Parallel tempering temperature in kT/e:             0.28426000
 Sweeps between replica exchange:                    0
 Sweeps between Grand-Canonical move:                100
 Sweeps between Cluster moves:                       0
 Wang-Landau method:                                 0 0
 Calculate the Wang-Landau method for atom type:     0
 Average type switch attempts per sweep:             0.00000000
 Number of Sweeps per pairlist update:               0
 Random number seed:                                 196150684
 Number of sweeps per writing out cluster info:      50000
 Sweeps between movie frames:                      50000

Reading topology...
Species:
Fields:11 0
Topology read of 1:        A (geotype: CPSC, 13) with parameters 1.33333 1.2 | 1.34695 0.3 | 60 5 | 0 

Topology read for molecule: A 
particle 1: 	1 
activity: 0.007000 
system:  A 172

Topology succesfully read. Generating pair interactions...
Generating GroupLists...

Reading configuration...
Box: (20, 20, 20)
Equilibration of maximum step sizes: 0 sweeps

Initializing energy functions...
Production run:  30000000 sweeps

sweep: 3000000 particles: 167 drift: -3.20142e-10
InteractionEnergy: -319.11, sweeps per hour: 3.80677e+06

sweep: 6000000 particles: 162 drift: -4.16549e-10
InteractionEnergy: -312.184, sweeps per hour: 3.60082e+06

sweep: 9000000 particles: 163 drift: -5.42059e-10
InteractionEnergy: -310.384, sweeps per hour: 3.66882e+06

sweep: 12000000 particles: 176 drift: -4.92946e-10
InteractionEnergy: -338.378, sweeps per hour: 3.76291e+06

sweep: 15000000 particles: 188 drift: 2.18279e-11
InteractionEnergy: -361.571, sweeps per hour: 3.7526e+06

sweep: 18000000 particles: 157 drift: 5.90262e-10
InteractionEnergy: -319.545, sweeps per hour: 3.64181e+06

sweep: 21000000 particles: 168 drift: 4.02906e-10
InteractionEnergy: -292.93, sweeps per hour: 3.56132e+06

sweep: 24000000 particles: 170 drift: 8.69477e-10
InteractionEnergy: -307.992, sweeps per hour: 4.20505e+06

sweep: 27000000 particles: 174 drift: 9.94078e-10
InteractionEnergy: -333.407, sweeps per hour: 3.94019e+06

sweep: 30000000 particles: 154 drift: 8.0945e-10
InteractionEnergy: -287.232, sweeps per hour: 3.92451e+06

Energy drift: 8.09450e-10 
Starting energy+pv: -6647.91454560 
Starting energy: -319.42091932 
Ending energy: -288.47365044 


******************************************************************************
*                               Moves Statistics                             *
******************************************************************************
Move                          Acc (%)   Rej (%)   Steps     
Insert move of type A         6.4       94        150061    
Remove move of type A         6.4       94        149939    
Average particles of type A         1.7e+02   
System:
A 154

Simulation time:2.9e+04 s
PairList generation time:0 %

DONE
Deallocating Conf...
Deallocating Sim...
Deallocating Topo...
